src/doc.rs

```diff
@@ -598,6 +613,13 @@ pub fn parse_ftd(
                 let value = resolve_ftd_foreign_variable(variable.as_str(), name)?;
                 s = state.continue_after_variable(variable.as_str(), value)?
             }
+            ftd::Interpreter::StuckOnTerm {
+                doc_index: index,
+                state: st,
+            } => {
+                // No config in fpm::FPMLibrary ignoring processing terms for now
+                s = st.continue_after_processing_terms(None, index)?;
```

Comment:
`unreachable()`. If something violates a fundamental assumption, do not silently eat it, and panic instead.